source/support/manage-secrets/index.html.md.erb

```diff
@@ -51,44 +45,93 @@ the `BEARER_TOKEN` environment variable:
     BEARER_TOKEN: ((report-bearer-token))
 ```
 
-To list all secrets for the GOV.UK Concourse team, run the following in the gds-cli:
+## List secrets
+
+To list secrets with names and values used by the GOV.UK account manager app, run the following in the gds-cli:
+
+```
+cf env govuk-account-manager
+```
+
+To list the names of the secrets used by all GOV.UK tools, run the following:
 
 ```
-$ gds cd secrets ls cd-govuk-tools
+gds cd secrets ls cd-govuk-tools
 ```
 
 [the daily statistics task]: https://github.com/alphagov/govuk-account-manager-prototype/blob/7e69b8cb8daec1e6e23d32374df3f8a1fa45b477/concourse/pipeline.yml#L249-L271
 [GOV.UK Notify API key]: https://github.com/alphagov/govuk-account-manager-prototype/blob/5443585c3d2dd5d399efc18f126f0bdf831036b5/concourse/tasks/deploy-to-govuk-paas.yml#L59
 
-## Rotating a secret
+## Add or change a secret
+
+1. Sign into the GOV.UK London PaaS by running the following in the gds-cli:
```

Comment:
For `gds cd` commands, the user needs to be signed into Concourse, not the PaaS.

Reply:
So for this section, the user only needs to be signed into Concourse.  For the next section, the user needs to be signed into both.

Reply:
@barrucadu thanks, I've amended, please take a look when you have a moment